packages/web-frontend/src/constants/constants.js

```diff
@@ -36,3 +36,7 @@ export const LOGIN_TYPES = {
   TOKEN: 'token',
   MANUAL: 'manual',
 };
+
+export const PENDING_PROJECT_ACCESS_TYPE = 'PENDING';
```

Comment:
Good idea adding these as conts. What do you think about making them a set. eg.
```suggestion
export const PROJECT_ACCESS_TYPES = {
  PENDING: 'PENDING',
  DENIED: 'DENIED',
  ALLOWED: 'ALLOWED'
```